charts/prometheus-snmp-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus SNMP Exporter
 name: prometheus-snmp-exporter
-version: 3.0.0
+version: 3.0.1
```

Please can you change this to be a major version change because it is changing an existing value and the new `prometheus-config-reloader` container takes different arguments etc than the current one.
```suggestion
version: 4.0.0
```

---

Hi @walker-tom, Thanks for the review & suggestions.
I've made the requested changes, please let me know if there is something that i need to change again.
thanks in advance.